spec.html

```diff
@@ -6211,6 +6222,8 @@
         1. Add _pending_ at the back of the Job Queue named by _queueName_.
         1. Return NormalCompletion(~empty~).
       </emu-alg>
+
+      <emu-note>For host environments consisting of multiple continents, when those host environments enqueue their own jobs, they should generally specify which continent's job queues they are enqueuing on.</emu-note>
```

Comment:
I am confused at how you could arrive at any other interpretation still so I don't see the note as necessary. Even externally, it seems obvious that the continent whose execution context you activate in order to call EnqueueJob will be the continent whose job queues you are appending to.